### DISCUSSION TITLE
New Resource Created, but it's not showing up in admin sidebar

### DISCUSSION THREAD

**Discussion by aboustayyef** [filamentphp/filament#1445] [Help] [answered]
I have 4 models in my app for which I created Filament resources. Only 3 of them are showing up in the admin sidebar, and for the life of me, I don't know why the 4th is not showing. How can I even troubleshoot this?

**Comment by danharrin** [comment, marked as answer]
Check the policy for the resource that is not showing - make sure `viewAny()` returns `true` if it is defined.

**Reply by aboustayyef** [reply]
That worked, Thanks!

**Reply by aboustayyef** [reply]
But now it shows only the table without the form... 🤔

**edit** : figured out the answer on my own. Thanks... the create method in the policy should also return true...
